### ISSUE TITLE
[YSQL] Error when SPLIT AT VALUES for an INCLUDE index

### ISSUE THREAD

**Issue by FranckPachot** [yugabyte/yugabyte-db#11880]
Jira Link: [DB-913](https://yugabyte.atlassian.net/browse/DB-913)
### Description

Bug miscounting the included columns when computing the split rows.
This:
```
create table franck (a int, b int, PRIMARY KEY(a asc)) split at values ((1));
create index franck1 on franck(b asc) include (a) split at values ((1));
```
errors in:
```
ERROR:  Illegal state: Number of split row values must be equal to number of primary key columns: 2 vs. 1
```
Bug in `PgCreateTable::BuildSplitRows()` identified https://yugabyte.slack.com/archives/CAR5BCH29/p1648142982737249?thread_ts=1648136189.746979&cid=CAR5BCH29

**Comment by tedyu** [comment]
Turns out that neither PgCreateTable::AddColumn() nor PgCreateTable::EnsureYBbasectidColumnCreated() adds the included column to range_columns_ field.

It seems to be a grammatical limit: putting the included column in the list of indexed columns should get over this limit.

**Comment by sidjha1** [comment]
Issue resolved by commit `8be9329`
